From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Thu, 9 Feb 2023 19:22:47 +0000
Subject: [PATCH] * builds/meson/parse_modules_cfg.py: Handle gxvalid and
 otvalid

These need a name mapping similar to what was done for other modules,
or linking will fail.
---
 builds/meson/parse_modules_cfg.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/builds/meson/parse_modules_cfg.py b/builds/meson/parse_modules_cfg.py
index 6030bb2098df..d48129fe72e1 100644
--- a/builds/meson/parse_modules_cfg.py
+++ b/builds/meson/parse_modules_cfg.py
@@ -97,8 +97,12 @@ def generate_ftmodule(lists):
 
     for module in lists["AUX_MODULES"]:
         if module in ("psaux", "psnames", "otvalid", "gxvalid"):
+            name = {
+                "gxvalid": "gxv",
+                "otvalid": "otv",
+            }.get(module, module)
             result += (
-                "FT_USE_MODULE( FT_Module_Class, %s_module_class )\n" % module
+                "FT_USE_MODULE( FT_Module_Class, %s_module_class )\n" % name
             )
 
     result += "/* EOF */\n"
